b9lab-content/2-what-is-cosmos/04-atom-staking.md

```diff
@@ -0,0 +1,184 @@
+# Getting ATOM and Staking
+
+<!--
+
+This content could be linked to from https://cosmos.network/learn/get-atom as a step-by-step.
+
+-->
+
+The [Cosmos Hub](https://hub.cosmos.network) is the first among equals of Cosmos blockchains. The ATOM is the native token of the Cosmos Hub, also called the staking token. You need some ATOM to be able to transact on the Cosmos Hub, or to participate in the proof-of-stake consensus. Because of its usefulness, there are markets that determine the ATOM price in common currencies.
```

Comment:
Early we say that you can use any token to transact. Which is true for now. It might change for a bit, where you need ATOM (eg. if we roll out an EIP1559), and then eventually it will change back (when we add support  for paying in non ATOM in an EIP1559 context)